.env.example

```diff
@@ -53,8 +53,8 @@ NEXTAUTH_SECRET=
 # Used for cross-domain cookie authentication
 NEXTAUTH_COOKIE_DOMAIN=.example.com
 
-# Remove this var if you don't want Cal to collect anonymous usage
-NEXT_PUBLIC_TELEMETRY_KEY=js.2pvs2bbpqq1zxna97wcml.oi2jzirnbj1ev4tc57c5r
```

Comment:
Did the key changed? Or this this unintentional?

Reply:
Yes, it is intentional. Since all events are sent from server-side, there's no need to expose the key to client rendering

Reply:
Sorry @vklimontovich I0ve meant the key VALUE itself changed. Was it regenerated?

Reply:
No, it's the same API, but different key (used for server-to-server communication with Jitsu)

Reply:
Gotcha!